src/parser/core/modules/GlobalCooldown.tsx

```diff
@@ -18,20 +18,34 @@ export class GlobalCooldown extends Analyser {
 		this.addEventHook('complete', this.onComplete)
 	}
 
-	public getEstimate() {
+	/**
+	 * Get the base recast time of the parsed actor's GCD cooldown group, in milliseconds.
+	 * The value returned from this function _may_ be an estimate - check {@link GlobalCooldown.isEstimated}
+	 * to see if it is.
+	 */
+	public getDuration() {
 		return this.speedAdjustments.getAdjustedDuration({duration: BASE_GCD})
 	}
 
+	/** Returns whether the GCD duration calculated by this module is an estimate. */
+	public isEstimated() {
+		return this.speedAdjustments.isAdjustmentEstimated()
+	}
+
 	private onComplete() {
+		const estimated = this.isEstimated()
+
 		this.statistics.add(new SimpleStatistic({
-			title: <Trans id="core.gcd.estimated-gcd">Estimated GCD</Trans>,
+			title: estimated
+				? <Trans id="core.gcd.estimated-gcd">Estimated GCD</Trans>
+				: <Trans id="core.gcd.gcd">GCD Recast</Trans>,
 			icon: this.data.actions.ATTACK.icon,
-			value: this.parser.formatDuration(this.getEstimate()),
-			info: (
+			value: this.parser.formatDuration(this.getDuration()),
+			info: estimated ? (
```

Comment:
Should we update the copy to be "...not available from FF Logs for players other than the player who uploaded the log. ..." ?

Reply:
yes. yes it would. bleh.

Reply:
Done. Given the report source system, I've split the message into a generic everyone-gets-this bit, and a more-specific line that can be changed per-source. Obv, just fflogs there for now.

I've used a new (`.v2`) ID because I changed the params to the message, which will cause i18n crashes if there's a mismatch. I'm thinking this should be something we do more often but I really need to work out the i18n story going forward, the tooling is barely functional.